converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Fhir/FhirService.cs

```diff
@@ -51,6 +57,30 @@ public Task<ImagingStudy> RetrieveImagingStudyAsync(Identifier identifier, Cance
         public Task<Endpoint> RetrieveEndpointAsync(string queryParameter, CancellationToken cancellationToken)
             => SearchByQueryParameterAsync<Endpoint>(queryParameter, cancellationToken);
 
+        /// <inheritdoc/>
+        public async System.Threading.Tasks.Task ValidateFhirService(CancellationToken cancellationToken)
+        {
+            using FhirResponse<CapabilityStatement> response = await _fhirClient.ReadAsync<CapabilityStatement>("metadata", cancellationToken);
+            var version = response.Resource.FhirVersion ?? throw new InvalidFhirServerException("FHIR server version cannot be validated, should be R4");
```

Comment:
We should create a custom exception with strings in resx file. Is that not what we are folowing dicom-cast?

Reply:
I added strings to the resx file. The exception thrown is a custom exception I added.